src/ledger_app.rs

```diff
@@ -0,0 +1,171 @@
+/*******************************************************************************
```

Comment:
also update https://github.com/crypto-com/chainlib-rs/blob/main/NOTICE

Reply:
should I remove this comment?